fedpydeseq2/core/utils/logging/logging_decorators.py

```diff
@@ -187,9 +287,150 @@ def log_shared_state_adatas(self: Any, method: Callable, shared_state: dict | No
             logger.debug(f"{adata_name} obsm keys : {list(adata.obsm.keys())}")
 
 
-def get_method_logger(self: Any, method: Callable) -> logging.Logger:
+def write_info_before_organisation_method(method: Callable):
+    """Append the information of the local step to a file.
+
+    Parameters
+    ----------
+    method : Callable
+        The method whose name will be logged.
+
+    Notes
+    -----
+    This function appends the following information to the specified file:
+    - The name of the method enclosed in `<name>` tags.
+    """
+    text_to_add = "<bloc>\n"
+    # Add name balise
+    text_to_add += f"<name>{method.__name__}</name>\n"
+    # Append the text to the file
+    file_path = get_workflow_file()
+    if file_path is not None and file_path.exists():
+        with open(file_path, "a") as file:
+            file.write(text_to_add)
+
+
+def write_info_after_organisation_method():
+    """Append the information of the local step to a file.
+
+    Notes
+    -----
+    This function appends the following information to the specified file:
+    - The name of the method enclosed in `<name>` tags.
+    """
```

Comment:
These two methods have exactly the same docstring, although they do different things